content/sdk/10.js/00.ethers/20.v6/01.features.md

```diff
@@ -79,23 +73,18 @@ and paymaster input `0x8c5a3445`:
 }
 ```
 
-## Encoding paymaster params
-
-While the paymaster feature by itself does not impose any limitations on values of the `paymasterInput`,
-the Matter Labs team endorses certain types of
-[paymaster flows](https://docs.zksync.io/build/developer-reference/account-abstraction/paymasters#built-in-paymaster-flows)
+## Encoding Paymaster Parameters
+While the paymaster feature itself does not impose any limitations on the values of `paymasterInput`, the Matter Labs
+team endorses certain types of [paymaster flows](https://staging-docs.zksync.io/build/developer-reference/account-abstraction#built-in-paymaster-flows)
```

Comment:
The URL here is incorrect, it's pointing to staging-docs. Use the production URL - docs.zksync.io